test/Microsoft.ML.Predictor.Tests/TestPredictors.cs

```diff
@@ -888,7 +888,7 @@ public void RegressorOlsTest()
         /// <summary>
         /// A test for ordinary least squares regression.
         /// </summary>
-        [Fact(Skip = "Need CoreTLC specific baseline update")]
+        [Fact]
```

Comment:
Can we enable the test above this too?  It is an `Ols` test...

Reply:
that tests fails the baseline comparison for 1/4896 predictions, in just Mac debug builds... 

 There are quite a bit of tests that are in this state. 
Just finished logging a proposal about how to deal with most of them, here: [https://github.com/dotnet/machinelearning/issues/619](https://github.com/dotnet/machinelearning/issues/619) 

---
In reply to: [206638252](https://github.com/dotnet/machinelearning/pull/611#discussion_r206638252) [](ancestors = 206638252)